### ISSUE TITLE
with torch.no_grad():

### ISSUE THREAD

**Issue by Longseabear** [anthonymlortiz/lcn#2]
why use  with torch.no_grad()?
With this implementation, no back propagation occurs in the mean and variance.
The batch normalization backpropagation backpropagate the mean and variance gradients.
Is this implementation correct?
